src/server/pkg/cmdutil/cobra.go

```diff
@@ -311,6 +311,18 @@ func (r *RepeatedStringArg) Type() string {
 	return "[]string"
 }
 
+// Hide marks all runnable subcommands as hidden and returns it
+// for method chaining.
+func Hide(cmd *cobra.Command) *cobra.Command {
```

Comment:
Just checking my understanding here, this is so if someone types `pachctl list` they don't get a huge autocomplete of like `repo, repos, job, jobs, file, files, ...`?